extensions/wikia/ArticleExporter/ArticleExporter.class.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+* ArticleExporter methods for getting and processing articles
+*
+* @author Lore team
+* @package ArticleExporter
+*/
+class ArticleExporter {
+    const SPACE_SEQUENCE_REGEXP = "/\s+/";
+
+    public function build($cityId, $ids) {
+        $articles = [];
+        foreach ( $ids as $id ) {
+            $article = $this->getArticle( $id );
+
+            if ( $article ) {
+                $articles[$id] = [
+                    'wikiId' => $cityId,
+                    'pageId' => $id,
+                    'revisionId' => strval( $article['parse']['revid'] ),
+                    'title' => $article['parse']['title'],
+                    'url' => $this->getPageUrl( $id ),
+                    'plaintextContent' => $this->getPlaintext( $article['parse']['text']['*'] ),
+                    'categories' => $this->getCategories( $article['parse']['categories'] ),
+                    'linkedPageTitles' => $this->getPageTitles( $article['parse']['links'] ),
+                    'updatedUtc' => $this->getUpdated( $article['parse']['revid'] ),
```

Comment:
Can we also add the namespace id? That'll help the caller to potentially filter pages that we might want to ignore such as the talk pages.

Reply:
Added namespace in a12368d.